### ISSUE TITLE
WIndows 10 Launching App

### ISSUE THREAD

**Issue by ericrobe** [writemonkey/wm3#172]
When I launch W3 on my windows 10 (64-bit) box it appears to hang. When I launch on my Windows 7 box it works as expected. 
- On Windows 10, I can see nw.exe running under 4 PIDs but the icon does not appear on the task bar & no window comes up. 
- I have downloaded the app twice (to ensure the first download was not corrupted). 
- I do not see any entries in the event log. 
- I have tried all compatibility modes without success. 
- I have unzipped W3 into my "My Documents" folder so there are no rights issues. 
- W2 launches without issue.
- Disabling Kaspersky & Malwarebytes does not resolve the issue

Looking for tips - I am out of ideas.

**Comment by writemonkey** [comment]
Hi, try this:

- launch wm3
- even without no wm on the screen press ctrl+shift+j
- wm dev tools should open, in console tab type `win.show()` and press enter
- or type `win.maximize()` and press enter

Does that brings wm up? It is a long shot, but please try it.
i.

**Comment by ericrobe** [comment]
ctrl+shift+j does not work. Is there a way to launch WM3 or the dev tools from a command line? 

Some more information: I am running Windows 10 Pro x64 build 10.0.16299. I am trying to launch the 64bit version of W3. Tonight I downloaded the 32-bit version of W3 and that launches as expected.

**Comment by ericrobe** [comment]
A quick followup - W3 64-bit version (3.0.4 beta) launches as expected on Windows 7 64-bit (Version 6.1.7601 SP1 Build 7601)

**Comment by ericrobe** [comment]
I copied the W3 64-bit version on my Windows 7 box to a USB stick and mounted it on my Windows 10 box. This copy of W3 will launch on my Windows 10 box.  I am going to assume either the two copies I downloaded from the W3 site where corrupted during the download or the windows 10 built in decompression app corrupted part of W3 during the decompression.

Either way, it looks like my problem was not a W3 problem - apologies.
